modules/installation-guide/partials/proc_configuring-che-hostname.adoc

```diff
@@ -29,15 +29,23 @@ $ {orch-cli} create {orch-namespace} {prod-namespace}
 +
 [subs="+quotes,attributes"]
```

Comment:
```suggestion
[source,shell,subs="+quotes,attributes"]
```

Reply:
Looks like we need shell here because the backslash after $ doesn't render at all in the published docs:
![image](https://user-images.githubusercontent.com/78564304/143435505-08efaa19-1302-49ae-af18-d58011daa8d8.png)